umbral/keys.py

```diff
@@ -313,76 +401,64 @@ def derive_privkey_by_label(self,
 
         key_material = HKDF(
             algorithm=hashes.BLAKE2b(64),
-            length=64,
+            length=64,  # TODO: Generalize for multicurve
```

Comment:
What exactly are you wanting to generalize for multicurve support? This is just deriving a key, then hashing it using whatever curve params were chosen in the call to `hash_to_curvebn` below.

Reply:
You're right! I'll remove the TODO.

Reply:
✅